.circleci/config.yml

```diff
@@ -11,6 +11,7 @@ executors:
   machine-executor:
     machine:
       docker_layer_caching: false
+      image: ubuntu-2004:202104-01
```

Comment:
This was necessary to use a newer version of the [Docker Compose file format](https://docs.docker.com/compose/compose-file/compose-versioning/#version-34)

For comparison, the default image is [ubuntu-1604:201903-01](https://discuss.circleci.com/t/default-machine-executor-image-update/29308) which just hit [end of life](https://computing.cs.cmu.edu/news/2020/eol-ubuntu-1604) as of April this year - because of this we were due for an update here anyway.